docs/architecture/adr-019-inflation-rate.md

```diff
@@ -0,0 +1,113 @@
+# ADR 019: Inflation rate change
+
+## Status
+
+Proposed
+
+## Changelog
+
+- 2023/3/31: initial draft
+
+## Context
+
+The inflation rate is decided to have a fixed rate with an initial value and a target value.
+Table below shows that how it looks like according to the made decision.
+
+| Initial Inflation |     8.00% |
+|------------------------|----------------------------|
+| Disinflation rate p.a |    10.00% |
+| Target Inflation (floor)  |  1.50% |
```

Comment:
```suggestion
| Name                     | Value  |
|--------------------------|--------|
| Initial Inflation        | 8.00%  |
| Disinflation rate p.a    | 10.00% |
| Target Inflation (floor) | 1.50%  |
```

[optional] https://marketplace.visualstudio.com/items?itemName=darkriszty.markdown-table-prettify
[proposal] to include a table header